crates/forge_main/src/ui.rs

```diff
@@ -511,6 +511,12 @@ impl<A: API + 'static, F: Fn() -> A> UI<A, F> {
                     crate::cli::IndexCommand::Delete { workspace_id, yes } => {
                         self.on_delete_workspace(workspace_id, yes).await?;
                     }
+                    crate::cli::IndexCommand::Logout => {
```

Comment:
- logout can't be at an index level
- complimentary `login` command should also exist.